state/model.go

```diff
@@ -95,6 +95,12 @@ type modelEntityRefsDoc struct {
 
 	// Applicatons contains the names of the applications in the model.
 	Applications []string `bson:"applications"`
+
+	// Volumes contains the IDs of the volumes in the model.
```

Comment:
They mean different things. A volume "in" the model means that it is tracked by the model. If it is "bound to" the model, it means that it dies when the model dies.